.gitattributes

```diff
@@ -0,0 +1,8 @@
+/driver-testsuite export-ignore
```

Comment:
not only on Travis but for anyone working on drivers, making contributing harder. And forcing to use source installs is also bad as it means that all other deps of the driver will also be installed from source, forbidding to use the cache (making builds slower)